src/components/modals/RoleModal.vue

```diff
@@ -107,6 +145,14 @@ ul.tokens li {
   }
 }
 
+.button.active {
```

Comment:
This is causing a blue underline when rendering:
![Bildschirmfoto 2021-01-22 um 21 41 15](https://user-images.githubusercontent.com/325521/105543848-936e7080-5cfa-11eb-9c14-26d6ead326e3.png)
Why not just use the `.townsfolk` CSS class for the active button?